EventFunction_minimalistic.m

```diff
@@ -0,0 +1,6 @@
+function [value,isterminal,direction] = EventFunction_minimalistic(~,y)
+%% [value,isterminal,direction] = EventFunction_minimalistic(~,y)
+value = imag(y(1)); % Ground at y == 0
```

Comment:
Could you add a comment here for some context? Eg.  what is the dimension of y? What are the elements? What is the `imag()` function doing?

Otherwise looks good!

Reply:
It looks like this function already exists in the main function, if so, then you can delete this file.